File: Herencia/ejercicio 5/Java/Gerente.java
class Gerente extends Empleado {
    public String departamento;
    public double bono;
    public Gerente(String nombre, String apellido, double salario, int añosAnti, String departamento, double bono) {
        super(nombre, apellido, salario, añosAnti);
        this.departamento = departamento;
        this.bono = bono;
    }
    @Override
    public double calcularSalario() {
        return super.calcularSalario() + bono;
    }
    public double getBonoGerencial() {
        return bono;
    }
}

File: Herencia/ejercicio 5/Python/Gerente.py
from Empleado import Empleado
class Gerente(Empleado):
    def __init__(self, nombre, apellido, salarioBase, añosDeAntiguedad, departamento, bono_gerencial):
        super().__init__(nombre, apellido, salarioBase, añosDeAntiguedad)
        self.departamento = departamento
        self.bono_gerencial = bono_gerencial
    def calcular(self):
        return super().calcular() + self.bono_gerencial